bounties_api/user/views.py

```diff
@@ -58,6 +59,15 @@ def get(self, request, public_address=''):
             {'nonce': user.nonce, 'has_signed_up': bool(user.email) and bool(user.name)})
 
 
+class DismissSignup(APIView):
+    permission_classes = [AuthenticationPermission, UserIDMatches]
+
+    def get(self, request, public_address=''):
+        user = User.objects.get(public_address=public_address.lower())
+        user.dismissed_signup_prompt = datetime.utcnow()
+        return JsonResponse(UserSerializer(request.current_user).data)
```

Comment:
Should we return the `user` here, which we are updating?

Reply:
I do want to return the user since I want the frontend to then update the model (it should be returned with the new field updated)